content/about/how-many-people-use-tor/contents.lr

```diff
@@ -0,0 +1,14 @@
+title: How many people use Tor? How many relays or exit nodes are there?
+---
+key: 8
+---
+description:
+
+The [Tor](https://www.torproject.org/) network is one of the largest deployed anonymity networks,
+consisting of [thousands](https://metrics.torproject.org/networksize.html) of volunteer-run 
+relays and [millions](https://metrics.torproject.org/userstats-relay-country.html) of users.
```

Comment:
"millions" is also too vague.

Reply:
@gusgustavo I did not use the exact numbers since it's a dynamic update. Would you prefer to timestamp the answer and give the exact numbers?
Please let me know